docs/zh/table_design/indexes/Bitmap_index.md

```diff
@@ -11,13 +11,13 @@ Bitmap 索引是一种使用 bitmap 的特殊数据库索引。bitmap 即为一
 
 Bitmap 索引能够提高指定列的查询效率。如果一个查询条件命中列，StarRocks 即可使用[前缀索引](./Prefix_index_sort_key.md)提高查询效率，快速返回查询结果。但是前缀索引的长度有限，如果想要提高一个非前缀索引列的查询效率，即可以为这一列创建 Bitmap 索引。
 
-Bitmap 索引一般适用于高基数列，基于列构建的 Bitmap 索引的选择性高，并且使用 Bitmap 索引后能筛选出较少数据行的场景。
+Bitmap 索引一般适用于低基数列，基于列构建的 Bitmap 索引的选择性高，并且使用 Bitmap 索引后能筛选出较少数据行的场景。
```

Comment:
Hi, the document is correct.
We want users to create a bitmap index for high cardinality columns.
Suppose creating a bitmap index for a sex column that just has male/female, half of the records will be read. In that case, the query performance can not be accelerated by the index.

Reply:
I looked at the 2.5 version documentation and Wikipedia, and it says bitmap is suitable for low cardinality. The example on your official website is also a low-base example.